src/main/java/edu/kis/powp/jobs2d/command/ComplexCommand.java

```diff
@@ -0,0 +1,23 @@
+package edu.kis.powp.jobs2d.command;
+
+import java.util.ArrayList;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class ComplexCommand implements DriverCommand{
+    ArrayList<DriverCommand> list;
```

Comment:
should be private
List type (attribute and in the constructor) would be sufficient and would not limit the implementation.